### PR TITLE
Possibility to determine event multiplicity with ITSrefit tracks

### PR DESCRIPTION
**Pull request by mfaggin**

### CODE REVIEW

File: PWGPP/ITS/AliAnalysisTaskSEImpParResSparse.cxx [hunk 1]
```diff
@@ -598,8 +642,24 @@ void AliAnalysisTaskSEImpParResSparse::UserExec(Option_t */*option*/)
...
+    highMult = (nTrks>500 ? kTRUE : kFALSE);
+    fNumTracksperEv->Fill(nTrks);
+    }
+    else{    // Consider the number of tracks with ITS refit only, avoiding the TPC pile-up
+        Int_t nTrks_ITSrefit=0;
+        AliVTrack* trk = 0;
+        for (Int_t it=0; it<nTrks; it++){ //start loop over tracks
+            trk = (AliVTrack*)event->GetTrack(it);
+            if(!trk) {continue;}
+            if(fIsAOD)  {if(((AliAODTrack*)trk)->GetStatus()&AliESDtrack::kITSrefit)    nTrks_ITSrefit++;}
```
Comment (fprino) [inline PWGPP/ITS/AliAnalysisTaskSEImpParResSparse.cxx]: Hi Mattia,

this if is not needed: you can call GetSTatus on the AliVTrack directly, no need to cast, this should speed up the execution.

Comment (mfaggin) [reply PWGPP/ITS/AliAnalysisTaskSEImpParResSparse.cxx]: Hi Francesco,

thanks for the comment. I implemented it in the new commit that I pushed. 
Apart from the piece of code that you highlighted yesterday, I modified another couple of lines where the same typecast was performed. Without it, the running time seems to be significantly reduced and the output (local test) does not seem to show differences with respect to the previous configuration.

**Comment by fprino** [comment]
+test

**Comment by fprino** [comment]
+1

**Pull request merged**